### PR TITLE
add cosine distance

### PR DESCRIPTION
**Pull request by harsham05**
Hi @chrismattmann 

This PR implements cosine distance metric based off @thammegowda Solr Cosine Distance implementation. Also adds @thammegowda to contributors

Thank you,
Harsha

### CODE REVIEW

File: vector.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import math
```
Comment (chrismattmann) [inline vector.py]: app ALv2 headers

File: cosine_similarity.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
+from tika import parser
```
Comment (chrismattmann) [inline cosine_similarity.py]: add ALv2 headers

File: vector.py (hunk 1 above)
Comment (harsham05) [inline vector.py]: Done

**Comment by harsham05** [comment]
@chrismattmann 
Added ALv2 headers.
Ready for review & merge :)

**Comment by chrismattmann** [comment]
LGTM!

**Pull request merged**